torch/random.py

```diff
@@ -65,9 +62,6 @@ def seed() -> int:
     if not torch.mps._is_in_bad_fork():
         torch.mps.manual_seed(seed)
 
-    if hasattr(torch, 'xpu') and not torch.xpu._is_in_bad_fork():
-        torch.xpu.manual_seed_all(seed)
-
```

Comment:
ditto

Reply:
We remove this code here, and add them back with PR about `Generator`.